CODE-OF-CONDUCT.md

```diff
@@ -1,5 +1,11 @@
-# Contributor Covenant Code of Conduct
-
+---
+title: "Contributor covenant code of conduct"
+linkTitle: "Code of conduct"
+weight: 10
+type: "docs"
+aliases:
+  - /contributing/code-of-conduct/
+---
```

Comment:
@evankanderson @psschwei Is the template for this one correct? Seems like it's coming via the website, which is not suitable for all repos?

Reply:
Well.... yeah, that doesn't look right. 

Not sure whether it's better [easier] to remove the automation or update the community doc (which would also entail some change for the COC doc for the website). 

All repos that need a COC for GA either already have it or have had a PR open to add it, so the simplest solution is probably to disable the automation for now. Sound good?

Reply:
I'm more concerned about the statement below regarding the copyright, as that might actively be wrong.

Reply:
@markusthoemmes so should we disable the COC from knobots and then close all the PRs that this one spawned?

Reply:
I opened a PR to disable : https://github.com/knative-sandbox/knobots/pull/118 , but we may have to lean on your superpowers again to close out the PRs